File: examples/react/src/token.ts
// Prints a token signed with the development key, as an identity provider would issue one at sign-in:
//   npm run token -- staff
import { devToken } from "./auth.ts";

const role = process.argv[2] === "staff" ? "staff" : "customer";
console.log(await devToken(role === "staff" ? "s1" : "u1", role));

File: examples/kotlin/src/main/kotlin/com/example/bookshop/Token.kt
package com.example.bookshop

/** Prints a token signed with the development key, as an identity provider would issue one: `./gradlew -q token --args=staff` */
fun main(args: Array<String>) {
    val role = if (args.firstOrNull() == "staff") "staff" else "customer"
    println(devToken(if (role == "staff") "s1" else "u1", role))
}
